GUI/GUIControlManager.h

```diff
@@ -119,7 +119,7 @@ friend class GUIControl;
 // Description:     Updates the GUI every frame
 // Arguments:       None.
 
-    void Update();
+    void Update(bool ignoreKeyboardEvents = false);
```

Comment:
Not really important, but this now has an argument that I guess should be in the method comment.